.github/workflows/ci.yml

```diff
@@ -340,8 +340,8 @@ jobs:
           platforms: linux/amd64,linux/arm/v7,linux/arm64
           push: true
           tags: ${{ steps.prepare.outputs.tags }}
-          cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
Does this need to be changed here too? https://github.com/paperless-ngx/paperless-ngx/blob/d2cde1c9352ff9297028dd816ae9d29462494554/.github/workflows/ci.yml#L320-L326

Reply:
Good catch! This step needs to be removed.